### DISCUSSION TITLE
Each tab group should have a separate set of essentials.

### DISCUSSION THREAD

**Discussion by thestrike72** [zen-browser/desktop#2798] [Ideas]
In my case, I have a tab group associated with my personal gmail account and one associated with school. I use my school tab group for a School workspace and a Work workspace, and I use my personal tab group for my other 4 workspaces. It would make sense for me to add GMail to essentials, but I use a different account for personal and work workspaces so I have to add it to essentials twice to have both accounts.

My claim is that for at least my use case, it makes sense to have a different set of essentials for each set of workspaces associated with a particular tab group.

**Comment by thestrike72** [comment]
Counterpoint to myself - this might not make sense for services such as spotify that should probably persist over all workspaces. Personally, I use the desktop app, but it's still a consideration.

**Comment by Madekuji** [comment]
I don't think "tab groups" is the right word to use here, but rather _Containers_ and _Container tabs_ (since _Tab Groups_ is a different upcoming feature altogether). But yes, coming from Arc, I have separate Personal and School _profiles_ (in Firefox/Zen, they're called _Containers_, not to be confused with Firefox profiles) and with the recent Essentials update, I expected the Essentials to still be different if the _Workspace_ (_Space_ in Arc) had a different Container applied to the entire Workspace.

In short,
**Expected behavior:** Essentials are global pinned tabs, like _Favorites_ in Arc, on a **per container** basis. Different container for workspace, different Essentials.
**Current behavior:** Essentials are global pinned tabs, NO exceptions. Even if a Workspace uses a different default Container, it still contains the same Essentials.

**Reply by vivianlauerbsu** [reply]
Exactly. I misspoke with tab containers vs. tab groups. It’s very confusing terminology. Profiles feels like a better abstraction.

**Reply by Madekuji** [reply]
Based on the Discord discussion from the developers, it seems like we're not gonna see this implemented and they would rather we use a mod ([SuperPins](https://zen-browser.app/mods/ad97bb70-0066-4e42-9b5f-173a5e42c6fc)) to have this fixed. It seems they prefer having Essentials as completely global despite differing default containers and Pinned Tabs to be workspace-specific, which is **not** how it works on Arc.

**Comment by nijin26** [comment]
I am also facing the same issue. It was good till the a17 version, this latest update making it a mess. They should provide an option to show the pinned tabs as icons instead of normal tabs ([Superpins](https://zen-browser.app/mods/ad97bb70-0066-4e42-9b5f-173a5e42c6fc) mod do the same, I am currently using it for now). Essentials doesn't make any sense to me (in my workflow), i don't want any tabs globally accessible in all of my workspace. Give the user a choice. :) Choose from what they have used to, instead of changing these on a sudden.

I have each workspace for there own purpose, so each of them have there own pinned tabs (it might be more than 5), then showing it as list is messy and hard to handle.

**Comment by Madekuji** [comment]
Update: It seems our request has been granted.
[https://www.reddit.com/r/zen_browser/comments/1gp4rxs/i_think_ive_found_the_sweet_spot_and_this_is_what/](https://www.reddit.com/r/zen_browser/comments/1gp4rxs/i_think_ive_found_the_sweet_spot_and_this_is_what/)
